src/CoreSpotlight/CSCompat.cs

```diff
@@ -1,18 +1,35 @@
 // Compatibility stubs
 
-#if !NET && IOS
-
 using System;
+using ObjCRuntime;
 
 namespace CoreSpotlight {
 
+#if !NET && IOS
 	partial class CSCustomAttributeKey {
 
 		[Obsolete ("Use .ctor(string)")]
 		public CSCustomAttributeKey () : this (String.Empty)
 		{
 		}
 	}
-}
+#endif
+
+#if !TV
+	public partial class CSSearchQueryContext {
 
+#if !NET
+		[Obsolete ("This property was removed. The getter always returns null and the setter throws and InvalidOperationException.")]
+#else
+		[UnsupportedOSPlatform ("ios16.1")]
+		[UnsupportedOSPlatform ("maccatalyst16.1")]
+		[UnsupportedOSPlatform ("macos13.0")]
 #endif
+		public string[] ProtectionClasses { 
```

Comment:
This must be virtual:

<img width="534" alt="Screen Shot 2022-10-17 at 07 16 55" src="https://user-images.githubusercontent.com/249268/196094847-a2cce15e-42de-4563-8c62-3ae39e9e7184.png">